### PR TITLE
Login screen

### PR DESCRIPTION
**Pull request by GBarron93**
Updated login screen.

### CODE REVIEW

**Comment by Andrew-Delgado** [comment]
## Good work just a few changes

1.  Set the color of a clicked button to white. 
2.  Move over the email and text buttons to the left since email is going to the next line.
3.  Make the email button inline with the Contacts first name.
Screenshot_1568404775

**Comment by Justin-Terry** [comment]
@GBarron93 @Andrew-Delgado 

The list items don't look like the ones in the screenshots. If you'd like to keep these list items please change the background color to be white and update the text color to be visible on the white background. 

Using a relative layout is fine as a root layout for the list items but since everything is placed relative to something else you are going to have problems like the email appearing on the wrong line if the screen is smaller that the one you are testing on in your virtual device. I would try laying this out as horizontal linear layout with a vertical linear layout for the text and a horizontal linear layout for the check boxes.

rootLinearLayout(horizontal)
    --> textLinearLayout(vertical)
        ----> TextView (name)
        ----> TextView (phone)
        ----> TextView (email)
    --> checkBoxLinearLayout (horizontal)
        ----> Checkbox
        ----> TextView (email)
        ----> Checkbox
        ----> TextView (text)

By structuring it this way you'll always have the two nested linear layouts next to each other, the TextViews stacked on the left and the CheckBoxes and TextViews next to each other on the right. Then just set the weights appropriately.

File: app/src/main/java/com/example/safehopper/ui/contacts/ContactsFragment.java [hunk 1]
```diff
@@ -1,51 +1,70 @@
...
-            @Override
-            public void onChanged(@Nullable String s) {
-                textView.setText(s);
-            }
-        });*/
+        recyclerView.setLayoutManager(new LinearLayoutManager(getActivity()));
+
         return root;
     }
+
```
Comment (Justin-Terry) [inline app/src/main/java/com/example/safehopper/ui/contacts/ContactsFragment.java:46]: Please change the initImageBitMaps function name to a name that relates to what the function does. There are no bitmaps being loaded by that function.

**Comment by Justin-Terry** [comment]
[Login Screen](https://trello.com/c/Zm2G5zTb/12-login-screen)

**Comment by Justin-Terry** [comment]
[Contacts Screen](https://trello.com/c/Bq7e68B3/13-contacts-screen)

**Pull request closed without merging**